_config.yml

```diff
@@ -4,8 +4,10 @@ permalink: /news/:year/:month/:day/:title/
 relative_permalinks: false
 markdown: kramdown
 highlighter: rouge
-site_env: prod
+site_env: demo
```

Comment:
The default should be production please.

Reply:
changed.